compiler-rt/lib/asan/asan_malloc_win.cpp

```diff
@@ -142,6 +142,26 @@ __declspec(noinline) void *_recalloc_base(void *p, size_t n, size_t elem_size) {
   return _recalloc(p, n, elem_size);
 }
 
+__declspec(noinline) void *_aligned_malloc(size_t alignment, size_t size) {
```

Comment:
Shouldn't the first parameter be `size` and the second `alignment`?

Reply:
Also, we need to actually intercept the system functions by adding these to `ReplaceSystemMalloc()`.

We should also remove the

```
// TODO(timurrrr): Might want to add support for _aligned_* allocation
// functions to detect a bit more bugs.  Those functions seem to wrap malloc().
```

comment now :-)